src/components/landscape-sidebar/geo-description-widget/geo-description-widget-styles.module.scss

```diff
@@ -12,6 +12,10 @@ $error-gif-size: 120px;
   position: relative;
 }
 
+.dummyBlurWorkaround {
+  position: fixed;
+}
```

Comment:
Maybe you can extract this to a separate component because now we are repeating this class three times in our app